src/app/home-page/widget/organizations/organizations.component.ts

```diff
@@ -0,0 +1,55 @@
+import { Component, OnInit } from '@angular/core';
+import { Observable, Subject } from 'rxjs';
+import { UserQuery } from 'app/shared/user/user.query';
+import { DefaultService } from 'app/shared/openapi';
+import { OrganizationUpdateTime } from 'app/shared/openapi/model/organizationUpdateTime';
+import { debounceTime, takeUntil } from 'rxjs/operators';
+import { formInputDebounceTime } from 'app/shared/constants';
+
+@Component({
+  selector: 'app-organizations',
+  templateUrl: './organizations.component.html',
+  styleUrls: ['./organizations.component.scss']
+})
+export class OrganizationsComponent implements OnInit {
```

Comment:
At first glance, this class looks a lot like entries.component.ts. Not sure if it's worth the effort, and it isn't that much code, but it seems like you could have a common base class.